src/frontend/services/signupService.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const signupService = async ({ name, email, password }) => {
+  try {
+    const {
+      data: { encodedToken },
+    } = await axios.post(`/api/auth/signup`, {
+      firstName: name,
+      lastName: "",
```

Comment:
Instead of doing this, why not just remove this field from the model?

Reply:
Updated